crates/sui-core/src/execution_scheduler/transaction_manager.rs

```diff
@@ -219,6 +225,9 @@ struct Inner {
 
     // Transactions that have all input objects available, but have not finished execution.
     executing_certificates: HashSet<TransactionDigest>,
+
+    // Transactions for which we do not have a digest or transaction yet
```

Comment:
I'm confused by the wording "or transaction" here? How could we have a transaction for which we don't have a transaction?